src/makeCalculator.js

```diff
@@ -4,7 +4,30 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+    result: 0,
+    add(a, b) {
+      return a + b;
```

Comment:
[CODE STYLE] - Instead of returning the sum directly, you should update the `result` property using the `+=` operator. This applies to all arithmetic methods. Quote from the checklist: "use arithmetic operators with assignment operator `=` if you want to add something to existing value".